extensions/ql-vscode/src/remote-queries/view/LastUpdated.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { CalendarIcon } from '@primer/octicons-react';
+import styled from 'styled-components';
+
+const Calendar = styled.span`
+  flex-grow: 0;
+  text-align: right;
+  margin-right: 0;
+`;
+
+const Duration = styled.span`
+  text-align: left;
+  width: 8em;
+  margin-left: 0.5em;
+`;
+
+type Props = { lastUpdated?: number };
+
+const LastUpdated = ({ lastUpdated }: Props) => (
+  Number.isFinite(lastUpdated) ? (
+    <>
+      <Calendar>
+        <CalendarIcon size={16} />
+      </Calendar>
```

Comment:
I suggest we change `CalendarIcon` to `RepoPushIcon`. We show the date from the last push to the repo, and this icon encapsulates that better.

```suggestion
import { RepoPushIcon } from '@primer/octicons-react';
import styled from 'styled-components';

const RepoPush = styled.span`
  flex-grow: 0;
  text-align: right;
  margin-right: 0;
`;

const Duration = styled.span`
  text-align: left;
  width: 8em;
  margin-left: 0.5em;
`;

type Props = { lastUpdated?: number };

const LastUpdated = ({ lastUpdated }: Props) => (
  Number.isFinite(lastUpdated) ? (
    <>
      <RepoPush>
        <RepoPushIcon size={16} />
      </RepoPush>
```